pyhive/presto.py

```diff
@@ -18,6 +18,7 @@
 import requests
 from requests.auth import HTTPBasicAuth
 
+from requests_kerberos import HTTPKerberosAuth, OPTIONAL
```

Comment:
above comment is obsolete assuming we just go with the `requests_options` approach.